packages/nuxt/src/app/composables/ssr.ts

```diff
@@ -26,19 +28,22 @@ export function useRequestHeaders (include?: any[]) {
   return headers
 }
 
+/** @since 3.9.0 */
 export function useRequestHeader(header: string) {
   if (import.meta.client) { return undefined }
   const event = useRequestEvent()
   return event ? getRequestHeader(event, header) : undefined
 }
 
+/** @since 3.2.0 */
 export function useRequestFetch (): typeof global.$fetch {
   if (import.meta.client) {
     return globalThis.$fetch
   }
   return useRequestEvent()?.$fetch as typeof globalThis.$fetch || globalThis.$fetch
 }
 
+/** @since 3.0.0 */
```

Comment:
When a function has multiple overloads, does each one need it's own annotation? I see that the last overload has a `@deprecated` annotation which does not affect the first.
Also, what about the case where a new overload is added in a newer version, can that have it's own `@since` annotation?

Reply:
I'm happy to stick with just one for now.